app/javascript/components/Projects/ProjectsList.tsx

```diff
@@ -161,10 +177,16 @@ export const ProjectsListRow: React.FC<{ project: Project }> = ({
       <Col xs={1} sm={1}>
         {orbs?.length > 0 ? (
           <Flex my={1} alignItems="center">
-            <Tags tags={formattedTags(orbs)} />
+            <Tags tags={formattedOrbs(orbs)} />
           </Flex>
+        ) : isKubernetes ? (
+          <XCircleIcon fill="red100" />
         ) : (
-          isKubernetes && <XCircleIcon fill="red100" />
+          ci_provider !== "circleci" && (
```

Comment:
I would love to see a helper function extracted here for this conditional logic - nested ternary expressions break my brain! 😝